apps/modernization-ui/src/styles/_uswds.scss

```diff
@@ -62,8 +62,10 @@ button.usa-button {
     svg {
         vertical-align: bottom;
     }
+    // display: flex;
```

Comment:
Unless you see a good reason to keep this, we should remove the comments here and the line below. Its just adding cruft.